.github/CODEOWNERS

```diff
@@ -81,6 +81,7 @@ exporter/syslogexporter/                                 @open-telemetry/collect
 exporter/tencentcloudlogserviceexporter/                 @open-telemetry/collector-contrib-approvers @wgliang @yiyang5055
 exporter/zipkinexporter/                                 @open-telemetry/collector-contrib-approvers @MovieStoreGuy @astencel-sumo @crobert-1
 
+extension/ackextension/                                  @open-telemetry/collector-contrib-approvers @atoulme
```

Comment:
We need more code owners from your side. This requires OTel org membership. @splunkericl I see you have made enough contributions to apply for membership. Can you submit an issue similar to https://github.com/open-telemetry/community/issues/1915?

Reply:
@zpzhuSplunk @splunkericl, while the component is in the development phase, I believe you should be able to add yourselves in the code owner list even without the Opentlemetry membership

Reply:
Sure we can.